Translations/zh-Hant.lproj/advanced.strings

```diff
@@ -51,28 +51,28 @@
 "2271.ibShadowedObjectValues[10]" = "128 GB Blu-ray XL (QL)";
 
 /* Class = "NSTextFieldCell"; title = "Volume name:"; ObjectID = "2343"; */
-"2343.title" = "映像名稱：";
+"2343.title" = "卷宗名稱：";
 
 /* Class = "NSTextFieldCell"; placeholderString = "custom name"; ObjectID = "2354"; */
 "2354.placeholderString" = "自訂名稱";
 
 /* Class = "NSTextFieldCell"; title = "You can create a ISO volume that will be read-only and compressed. ISO volumes can be only created from a folder.\n\nSet a volume name to this ISO or use the default name based on the files you compress."; ObjectID = "2363"; */
-"2363.title" = "您可以建立唯讀的壓縮 ISO 映像。ISO 映像只可以壓縮檔案夾。\n\n設定映像名稱或根據壓縮檔案產生名稱。";
+"2363.title" = "您可以建立唯讀的壓縮 ISO 映像。ISO 映像只可以壓縮檔案夾。\n\n設定 ISO 的卷宗名稱，或根據壓縮檔案產生名稱。";
```

Comment:
Here also volume name, check https://github.com/aonez/Keka/pull/769#discussion_r563692462

Reply:
In this line it says that you can set a custom volume name and if not it will be set like the source file name.
For example if you compress a folder called "Hello" and do not set a custom volume name, the ISO image will be named "Hello.ISO" and once mounted it's volume name will be "Hello".

Reply:
Oh, I finally know why you wonder “卷宗“ is about “file”. That's the fault of Google Translate.

In the Chinese (Traditional) terminology released by Apple, “volume” is “卷宗”. Actually there are a more exact translation called “磁碟區” which is from Microsoft. But for the consistently, I used the Apple's term.

I recommended you finding the terminology with the terminology database from [Microsoft](https://www.microsoft.com/zh-tw/language?rtc=1) or [Apple (Search “Glossaries - macOS”)](https://developer.apple.com/download/more/), which is more professional and more accurate, and is often used by us a localization translator.

Reply:
> Oh, I finally know why you wonder “卷宗“ is about “file”. That's the fault of Google Translate.
> 
> In the Chinese (Traditional) terminology released by Apple, “volume” is “卷宗”. Actually there are a more exact translation called “磁碟區” which is from Microsoft. But for the consistently, I used the Apple's term.
> 
> I recommended you finding the terminology with the terminology database from [Microsoft](https://www.microsoft.com/language) or [Apple (Search “Glossaries - macOS”)](https://developer.apple.com/download/more/), which is more professional and more accurate, and is often used by us a localization translator.

By the way, we often use “檔案” for “file”.

Reply:
Ok!!! Thanks a lot @pan93412 for the explanation. I wish I know Korean 😉

Reply:
> Ok!!! Thanks a lot @pan93412 for the explanation. I wish I know Korean 😉

It is Chinese, not Korean! There are a lot of difference! 😆

Reply:
I was about to change that 😂 sorry, mean Traditional Chinese 😂😂🙇🏻‍♂️

Reply:
> I was about to change that 😂 sorry, mean Traditional Chinese 😂😂🙇🏻‍♂️

That is not a big deal! I'm glad to discuss these with you :)